pkgs/by-name/lm/lm_sensors/package.nix

```diff
@@ -2,41 +2,63 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  fetchpatch,
   bash,
   bison,
   flex,
   which,
   perl,
+  rrdtool,
   sensord ? false,
-  rrdtool ? null,
 }:
 
-assert sensord -> rrdtool != null;
+let
+  version = "3.6.0";
+  tag = lib.replaceStrings [ "." ] [ "-" ] version;
```

Comment:
Should this be "V3-6-0" rather than just "3-6-0"?

I'm getting the following error when trying to build from source:

```
trying https://github.com/lm-sensors/lm-sensors/archive/refs/tags/3-6-0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
```

Reply:
Thanks for the catch! Filed #375622